guides/include-cleaner.md

```diff
@@ -25,13 +16,51 @@ This means:
 This is opinionated, and stricter than "will it compile". It implies adding
 `#includes` that the compiler does not strictly need. Some `#includes` that
 satisfy the compiler should be replaced with more specific ones.
+If your codebase does not broadly aim to follow this style, you should disable
+these features.
 
-There can be exceptions to these rules. Nevertheless, if your codebase does not
-broadly aim to follow this style, you should disable this feature.
+## Show which headers are used
+
+{:.v16}
+
+Hovering on a name will show you which header provides that symbol:
+![Symbol hover showing provider in VSCode](../screenshots/symbol_header.png)
+
+Hovering on an `#include` will show which of its symbols are used in the file.
+![#include hover showing symbols in VSCode](../screenshots/header_symbols.png)
+
+Using "find references" you can navigate to these uses.
+
+## "Unused include" warning
+
+{:.v14}
+
+`#include`ing headers that are not used can confuse readers, slow down build
+times, and make code harder to refactor. clangd can warn about this:
+
+![Unused include warning in VSCode](../screenshots/unused_include.png)
+
+Often, these `#include` directive has no effect and can simply be removed.
+This isn't always the case: the analysis makes assumptions about code style.
+
+## "Missing include" warning
+
+{:.v16}
+
+Similarly, using symbols from a header without `#include`ing it is brittle and
+hides dependencies. clangd can show such uses:
+
+![Missing include warning in VSCode](../screenshots/missing_include.png)
+
+Generally clangd can suggest the correct `#include` to add here. In some
+cases there might be multiple options.
```

Comment:
AFAIK we don't surface multiple options in `clangd`. It's an implementation detail not visible to user.

Reply:
Right - by "options" I meant there might be several possibilities, and the one clangd recommends might not be the one you want! Rephrased